### ISSUE TITLE
button.vidbttn does not have an accessible name

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#1447]
## Problem

The homepage vertical video buttons don't have an accessible name to identify it beyond "button."

## Proposed Solution
Set an aria-label based on the video being displayed: "Play/pause video about ___"?
